science/memesuite/Portfile

```diff
@@ -3,8 +3,7 @@
 PortSystem          1.0
 
 name                memesuite
-version             5.1.1
-revision        2
+version             5.4.1
```

Comment:
`revision` should be kept (and set to 0 and properly indented).

Reply:
(Also, we're missing the git handles of two of the maintainers.)

Reply:
Fixed